handbook/customer-success/README.md

```diff
@@ -0,0 +1,228 @@
+  - Slack, if we have a shared channel.
+  - email, using the email address from the calendar invite.
+  - LinkedIn, send a direct message.
+  - an alternative date and time. Suggest two to three options from which the customer can choose.
+    - Confirm that contact information is accurate and that the customer can receive and access meeting invites.
+
+### Generate a trial license key
+1. Fleet's self-service license key creator is the best way to generate a proof of concept (POC) or renewal/expansion Fleet Premium license key. 
+    - [Here is a tutorial on using the self-service method](https://www.loom.com/share/b519e6a42a7d479fa628e394ee1d1517) (internal video)
+    - Pre-sales license key DRI is the Director of Solutions Consulting
+    - Post-sales license key DRI is the VP of Customer Success
+
+2. Legacy method: [create an opportunity issue](https://github.com/fleetdm/confidential/issues/new/choose) for the customer and follow the instructions in the issue for generating a trial license key.
+
+## Managed Cloud 
+
+Managed Cloud is our hosted Fleet offering for Fleet Premium and Fleet Ultimate customers. All managed Fleet instances receive 24/7 monitoring and on-call support from Customer Success. 
+
+### 24/7 on-call
+
+The 24/7 on-call (aka infrastructure on-call) responsibility happens in shifts of one week.
+
+First responders:
+
+- Robert Fairburn
+- Kathy Satterlee
+
+Escalations (in order):
+
+- Eric Shaw (fleetdm.com)
+- Zay Hanlon
+- Luke Heath
+- Mike McNeil
+
+Expected response times: during business hours, 1 hour. Outside of business hours <4 hours.
+
+The first responder on-call for Managed Cloud will take ownership of the @infrastructure-oncall alias in Slack first thing Monday morning. The previous week's on-call will provide a summary in the #g-customer-success Slack channel with an update on alarms that came up the week before, open issues with or without direct end-user impact, and other issues to keep an eye out for.
+
+Escalation of alarms will be done manually by the first responder according to the escalation contacts mentioned above. A [suspected outage issue](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23outage%2C%23g-cx%2C%3Arelease&projects=&template=outage.md&title=Suspected+outage%3A+YYYY-MM-DD) should be created to track the escalation and determine root cause. 
+
+All infrastructure alarms (fleetdm.com and Managed Cloud) will go to #help-p1.
+
+When the current 24/7 on-call engineer is unable to meet the response time SLAs, it is their responsibility to arrange and designate a replacement who will assume the @oncall-infrastructure Slack alias.
+
+#### Runbooks
+
+Use [on-call runbooks](https://github.com/fleetdm/confidential/tree/main/infrastructure/runbooks#readme) to guide your response. Runbooks provided detailed, step-by-step instructions to quickly and effectively respond to and resolve most 24/7 on-call alerts. 
+
+## Responsibilities
+
+#### First-line response
```

Comment:
This is not a valid responsibility heading.  Imperative mood verb phrase only.  H3 only. (`###`)

Reply:
fyi @lukeheath @noahtalerman @alexmitchelliii @JoStableford  within "Responsibilities", keep it flat and use only h3s in the imperative mood (i.e. `### Do a thing`)

https://fleetdm.com/handbook/company/leadership#outline-of-departmental-page-structure